README.rst

```diff
@@ -158,5 +179,938 @@ using the ``gridded_perms_of_length`` method.
        102: (0, 0), (0, 0), (1, 1)
 
 There are numerous other methods and properties. Many of these specific
-to the ``tilescope`` algorithm, discussed in `Christian Bean’s PhD
-thesis <https://skemman.is/handle/1946/31663>`__.
+to the ``TileScope`` algorithm, discussed in `Christian Bean’s PhD
+thesis <https://opinvisindi.is/handle/20.500.11815/1184>`__. For the remainder
+of this readme we will focus on the ``TileScope`` algorithm.
+
+The TileScope algorithm
+=======================
+
+
+Using TileScope
+---------------
+
+If you've not installed ``tilings`` yet then go ahead and do this first by
+pip installing ``tilings``:
+
+.. code:: bash
+
+       pip install tilings
+
+Once done you can use the ``TileScope`` algorithm in two ways, either directly
+by importing from the ``tilings.tilescope`` module which we will discuss in
+greater detail shortly, or by using the ``TileScope`` command line tool.
+
+The command line tool
+---------------------
+
+First, check the help commands for more information about its usage.
+
+.. code:: bash
+
+       tilescope -h
+       tilescope spec -h
+
+To search for a combinatorial specification use the subcommand
+``tilescope spec``, e.g.
+
+.. code:: bash
+
+       tilescope spec 231 point_placements
+
+It will always try to solve for the generating function, although in some
+cases you will come across some not-yet-implemented features; for more information
+please join us on our `Discord server <https://discord.gg/ySJD6SV>`__,
+where we'd be happy to talk about it!
+
+For more information on the packs, skip ahead to the strategies section.
```

Comment:
"The last argument above `point_placements` is a strategy pack, which we explain in detail in the StrategyPacks section"

Reply:
edited to 
"The ``point_placements`` argument above is a strategy pack, which we explain in
more detail in the ``StrategyPacks`` section."